### PR TITLE
[dv,chip] Fix PR#16581 comments

### PR DESCRIPTION
**Pull request by ariel9678**
This PR address the comments mentioned in PR #16581. 
Signed-off-by: Ariel Cohen <ariel.cohen@nuvoton.com>

### CODE REVIEW

File: sw/device/tests/sim_dv/ast_clk_rst_inputs.c [hunk 1]
```diff
@@ -204,15 +205,22 @@ void init_units() {
  *  configure adc module
  */
 static void configure_adc_ctrl(const dif_adc_ctrl_t *adc_ctrl) {
+  int WakeUpTimeAonCycles;
+  int PowerUpTimeAonCycles;
```
Comment (matutem) [inline sw/device/tests/sim_dv/ast_clk_rst_inputs.c:209]:
```suggestion
  int wake_up_time_aon_cycles;
  int power_up_time_aon_cycles;
```
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#naming

**Review by matutem — APPROVED** [review]
Please update names per style guide as indicated, otherwise LGTM.

File: sw/device/tests/sim_dv/ast_clk_rst_inputs.c (hunk 1 above)
Comment (ariel9678) [reply sw/device/tests/sim_dv/ast_clk_rst_inputs.c]: Fixed.

Comment (ariel9678) [reply sw/device/tests/sim_dv/ast_clk_rst_inputs.c]: Fixed.

Comment (ariel9678) [reply sw/device/tests/sim_dv/ast_clk_rst_inputs.c]: @matutem I see that the kokoro fails, but I don't see why.
can you  please help me to understand what is the reason for this fail.

Comment (msfschaffner) [reply sw/device/tests/sim_dv/ast_clk_rst_inputs.c]: that was an intermittent error - I reran kokoro and it is passing now.

**Comment by msfschaffner** [comment]
Thanks @ariel9678!

**Pull request merged**